### ISSUE TITLE
Investigate why failing to upload package to pypi doesn't mark travis job failed

### ISSUE THREAD

**Issue by grzanka** [pytrip/pytrip#46]
See i.e. https://travis-ci.org/pytrip/pytrip/jobs/140848166

Travis failed to upload, twine error message:

```
+twine upload -r pypi dist/pytrip98-0.3.0-py2.py3-none-any.whl
Uploading distributions to https://pypi.python.org/pypi
Uploading pytrip98-0.3.0-py2.py3-none-any.whl
HTTPError: 400 Client Error: Invalid classifier "License :: OSI Approved :: GPL License" for url: https://pypi.python.org/pypi
```

**Comment by grzanka** [comment]
Looking at https://docs.travis-ci.com/user/customizing-the-build

> The exit code of after_success, after_failure, after_script and subsequent stages do not affect the build result. However, if one of these stages times out, the build is marked as a failure.

**Comment by grzanka** [comment]
Not yes, fixed, failing due to some numpy problem, see https://travis-ci.org/pytrip/pytrip/jobs/155154459. Already mentioned in #44

**Comment by grzanka** [comment]
Fixed in PR #68
